core/teams/models.go

```diff
@@ -0,0 +1,52 @@
+package teams
+
+import vgrand "code.vegaprotocol.io/vega/libs/rand"
+
+const teamIDLength = 64
+
+type TeamID string
+
+type Party string
+
+type Team struct {
+	ID TeamID
+
+	Leader    Party
+	Followers []Party
```

Comment:
I would name those as they are in the spec to be fair.

Reply:
Sure.